Spigot-Server-Patches/0505-No-Tick-view-distance-implementation.patch

```diff
@@ -164,6 +164,12 @@ index 0f303be3c3257548d1888ddbb575ba69ba12d0b8..7b2a3287ce8d296d29cbef45322a4699
 +        // Paper start - per player view distance
 +        // there can be potential desync with player's last mapped section and the view distance map, so use the
 +        // view distance map here.
++        if (chunkMap.useSlowerMethodForBroadcasts) {
```

Comment:
as the code states we are not using the old code because it desyncs. last thing we need are reports of issues thanks to this happening.